lawnchair/src/app/lawnchair/qsb/providers/Yandex.kt

```diff
@@ -0,0 +1,20 @@
+package app.lawnchair.qsb.providers
+
+import android.content.Intent
+import com.android.launcher3.R
+
+data object Yandex : QsbSearchProvider(
+    id = "yandex",
+    name = R.string.search_provider_yandex,
+    icon = R.drawable.ic_yandex,
```

Comment:
As said above, we can change this to be the following:


```suggestion
    icon = R.drawable.ic_yandex,
    themedIcon = R.drawable.ic_yandex_tinded,
```

Reply:
fixed